pgxn/neon/libpqwalproposer.c

```diff
@@ -74,7 +74,7 @@ walprop_connect_start(char *conninfo, char *password)
 	if (password)
 	{
 		keywords[n] = "password";
-		values[n] = neon_auth_token;
+		values[n] = password;
```

Comment:
I think it's ok. This is a bugfix, because right now `walprop_connect_start(char *conninfo, char *password)` has weird behaviour – we check `if (password)` but don't use `password` anywhere else.

Reply:
Sounds good